configure.ac

```diff
@@ -1,10 +1,8 @@
 AC_PREREQ(2.67)
-AC_INIT([ARPACK-NG],[3.6.0],[https://github.com/opencollab/arpack-ng/issues/],[arpack-ng],[https://github.com/opencollab/arpack-ng/])
+AC_INIT([ARPACK-NG],[3.6.3],[https://github.com/opencollab/arpack-ng/issues/],[arpack-ng],[https://github.com/opencollab/arpack-ng/])
```

Comment:
```suggestion
AC_INIT([ARPACK-NG],[3.7.0],[https://github.com/opencollab/arpack-ng/issues/],[arpack-ng],[https://github.com/opencollab/arpack-ng/])
```

Reply:
Should we bump the soname while we're at it?  If so, to what?

Reply:
Yes, please

Reply:
If I'm understanding your comment on #140 correctly, the dbgini and staini interfaces were added in version 3.6.3.  Is that right?  If so, then that version should have been updated to something like libarpack.so-2.1.0 (rather than -2.0.1, or not at all).